### ISSUE TITLE
[BasicUI] Icons & font MaterialIcons-Regular are missing

### ISSUE THREAD

**Issue by htreu** [eclipse/smarthome#3523]
In BasicUI some of the icons do not show up, i.e. the group navigation (arrow left/right) or the color picker items. In addition the requested font MaterialIcons-Regular is not available from the web/fonts folder. I wonder why this ever worked because the files for this font never were part of the repository according to git history.

![image](https://cloud.githubusercontent.com/assets/1729268/26488119/75ff7cea-4202-11e7-835b-b98f0c479302.png)

I can see that @resetnow did changes to material-icons.css which indicate that this should have been working some day with 072e76c7be6efadd93821fba17b82516b3265c83 what am I missing here?

When I revert 072e76c7be6efadd93821fba17b82516b3265c83 and put current version of MaterialIcons-Regular.woff2 in the web/fonts folder everything works well.

Tested in Chrome/Safari/Firefox on macOS.

**Comment by kaikreuzer** [comment]
@htreu I didn't look into it, but just want to mention that all looks good in the very latest openHAB distro build nonetheless:

<img width="1058" alt="screen shot 2017-05-26 at 11 25 37" src="https://cloud.githubusercontent.com/assets/3244965/26489046/112d9b36-4206-11e7-81e0-06e06c16352c.png">

**Comment by htreu** [comment]
In openHAB the fonts are provided by a fragment, see https://github.com/openhab/openhab-core/tree/master/bundles/org.openhab.ui.basicui/patch/web/fonts

**Comment by htreu** [comment]
It seems the font provided by the above mentioned fragment in openHAB is different from the MaterialIcons-Regular font which is already part of ESH in PaperIU.

I suggest to add the PaperUI version of the font to BasicUI (we already have a CQ for this version) and remove the patch here.

@resetnow is there anything to be said against this?

**Comment by resetnow** [comment]
IIRC the mentioned commit contains changes​ caused by using the same Material Icons font as PaperUI. I'll check if it's correct. I have to note that for me the fonts provided by openHAB overlay work well.

**Comment by htreu** [comment]
@resetnow yes the overlay works well. But in ESH the font is completely missing and the version from PaperUI (which would be nice to use since it already has CQ) uses the "old" codes.

**Comment by kaikreuzer** [comment]
@resetnow & @htreu Any suggestion how to proceed here? Could we simply switch to the version that is also used by the Paper UI?

**Comment by resetnow** [comment]
So the proposal here is to remove fonts from openHAB overlay and copy the ones from PaperUI to BasicUI ESH extension? If this is a matter of making ESH bundle work on its own, could we also just move fonts from overlay to ESH?

**Comment by htreu** [comment]
For ESH the solution is to use the PaperUI font (for which we have a CQ approval) also in BasicUI. This means reverting https://github.com/eclipse/smarthome/commit/072e76c7be6efadd93821fba17b82516b3265c83 so BasicUI uses the old codes again. This would also mean the overlay in openHAB must be updated or dropped.

**Comment by resetnow** [comment]
Why can't we just move existing fonts to ESH?

**Comment by htreu** [comment]
We would have two different versions of the same font which is confusing at least. Also one of the fonts (the openHAB overlay) is not approved by the Eclipse IP Team. @kaikreuzer do you second this?

**Comment by resetnow** [comment]
>is not approved by the Eclipse IP Team

I wasn't aware of that.

>two different versions of the same font 

I don't remember why exactly I changed these font codes, but odds are it's for Internet Explorer compatibility. I'll check a bit later.

**Comment by resetnow** [comment]
>is not approved by the Eclipse IP Team

But there was a CQ?

https://github.com/eclipse/smarthome/pull/536

>Material Design iconset removed as that CQ is still **pending**

**Comment by htreu** [comment]
Yes, I did read the conversation you and @kaikreuzer had back in the days. My understanding is this: Your initial work on BasicUI got merged w/o the iconset (see https://github.com/eclipse/smarthome/pull/536#issue-118621055) and then @kaikreuzer did the fragment in openHAB providing the iconset (see https://github.com/eclipse/smarthome/pull/411#issuecomment-159585499). Unfortunately the iconset from openHAB did not match your initial version and you did a refactoring of HMTL codes (see https://github.com/eclipse/smarthome/pull/785). Now we have ESH BasicUI using codes matching the openHAB fragment and a different font version from PaperUI (matching your initial font which did not make it).
My suggestion is still to revert your commit and go back to the original codes + using the font from paperUI. The font in the openHAB fragment may then be dropped.
